tools/scaffolding/templates/gateway/Makefile

```diff
@@ -41,3 +41,7 @@ yarn-install: yarn-ensure
 .PHONY: yarn-ensure
 yarn-ensure:
 	@$(SHELL) $(TOOLS_MODULE_DIR)/install-yarn.sh
+
+.PHONY: frontend-dev # Start the frontend in development mode.
+frontend-dev: yarn-ensure
+	yarn --cwd frontend install --frozen-lockfile && yarn --cwd frontend start
```

Comment:
```suggestion
frontend-dev: yarn-install
	yarn --cwd frontend start
```

Reply:
Sure, that's copypasta from the main Makefile, so I didn't look at it too closely. If you think that it's un-needed duplication I'm OK with removing it. However just for posterity can you describe why these two Makefiles should diverge on this point? Or should I DRY this in the main Makefile as well?

Reply:
Yeah this is a bit wonky, we actually don't have similar targets in each Makefile though they have similar names which makes it really confusing. I'm going to merge this as-is and true up separately.